docs/developers/docs/development_setup.md

```diff
@@ -133,19 +142,17 @@ point. To compile ReScript code again (if you've made changes), you can either d
 
     # Recompile, and then watch for changes
     yarn run re:watch
+## Run Webpack Dev Server
+
+Start the Webpack Dev Server on another tab or window:
 
+    yarn run wds
 ## Start the Rails server
 
 With `webpack-dev-server` running, start the Rails server:
 
     bundle exec rails server
```

Comment:
The Rails server actually needs to be started _before_ WDS because running the Rails server is what generates the `locales.json` file necessary for Webpack compilation.